packages/app/obojobo-document-json-parser/__tests__/__snapshots__/jsonToXml.test.js.snap

```diff
@@ -0,0 +1,468 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`JSON to XML Test Converts assessment.json to assessment.xml 1`] = `
+"<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
+<ObojoboDraftDoc>
+	<Assessment title=\\"Assessment\\" review=\\"always\\" rubric=\\"[object Object]\\" attempts=\\"5\\" id=\\"my-assessment\\">
```

Comment:
There are a couple of `[object Object]` attributes I saw - Like `rubric` and `actions` on ActionButton.